scripts/files.sh

```
@@ -1203,11 +1131,14 @@ YAML
       TZ: "${TIMEZONE}"
     volumes:
       - "${ARR_DOCKER_DIR}/gluetun:/gluetun"
-    ports:
-      - "${LOCALHOST_IP}:${GLUETUN_CONTROL_PORT}:${GLUETUN_CONTROL_PORT}"
-      - "${LAN_IP}:${QBT_PORT}:${QBT_INT_PORT}"
 YAML
 
+  local -a split_gluetun_ports=(
+    '\${LOCALHOST_IP}:${GLUETUN_CONTROL_PORT}:${GLUETUN_CONTROL_PORT}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)